website/data/cloud-docs-nav-data.json

```
@@ -297,7 +297,8 @@
         "title": "Organizations",
         "routes": [
           {"title": "Overview","path": "users-teams-organizations/organizations"},
-          { "title": "VCS Status Checks", "path": "users-teams-organizations/organizations/vcs-status-checks" }
+          { "title": "VCS Status Checks", "path": "users-teams-organizations/organizations/vcs-status-checks" },
+          { "title": "VCS Speculative Plan Management", "path": "users-teams-organizations/organizations/vcs-speculative-plan-management" }
```

```suggestion
          { "title": "Automatically cancel plan-only runs", "path": "users-teams-organizations/organizations/vcs-speculative-plan-management" }
```
Just to match the page title.